src/app/tests/TestSceneTable.cpp

```diff
@@ -1046,7 +1065,7 @@ TEST_F(TestSceneTable, TestHandlerFunctions)
         MockCCPairs[5].attributeID = MockAttributeId(kEnhancedColorMode);
         MockCCPairs[5].valueSigned8.SetValue(static_cast<int8_t>(-2)); // will cap to -1
         MockCCPairs[6].attributeID = MockAttributeId(kColorLoopActiveId);
-        MockCCPairs[6].valueSigned16.SetValue(static_cast<int16_t>(0x7FFF)); // will cap to 0x7FFE in int16
+        MockCCPairs[6].valueSigned16.SetValue(static_cast<int16_t>(0x7FFE)); // will cap to 0x7FFD in int16
```

Comment:
I don't get any of this.  What's capping to 0x7FFD?  The max int16 value is 0x7FFF.

Reply:
The maximum is forced in the mock attribute declaration. The goal was to test a value cap to max when the input value is above the limit but lower than a NULL representation. The cap is declared by `minMaxValue16S` above.